lib/slide.js

```diff
@@ -6,22 +6,45 @@ class Slide {
         this.powerPointFactory = args.powerPointFactory;
         this.name = args.name;
         this.externalObjectCount = args.externalObjectCount || 0;
+        this.parentContainer = args.parentContainer;
 
-        this.Images = [];
+        this.Objects = [];
```

Comment:
this name is a bit unclear for me because it could be confused with the native JavaScript `Object` what about `Elements` or something else?

Reply:
Also why is it capitalized? If it's just an array with no special attributes i'd expect it to be lowercase.

Reply:
OK, I'll change it to elements. Capital by mistake.